src/App.js

```diff
@@ -1,11 +1,25 @@
-import React, { useState, useEffect } from 'react';
+import PubSub from 'pubsub-js';
+import React, { useEffect, useState } from 'react';
 import './App.css';
 
+const MFE_TOPIC = 'MFE Topic';
+const topicSubscriber = (msg, data) => {
+  console.log("Received message in dogs MFE");
+  console.log("msg: ", msg);
+  console.log("data: ", data);
+};
+PubSub.subscribe(MFE_TOPIC, topicSubscriber);
```

Comment:
This subscription should be done only once, at app init.
Otherwise, the messages would be sent multiple times unnecessary.